.github/workflows/s3-tables-tests.yml

```
@@ -194,6 +194,73 @@ jobs:
+
+      - name: Set up Go
+        uses: actions/setup-go@v6
+        with:
+          go-version-file: 'go.mod'
+        id: go
+
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v4
+
+      - name: Pre-pull Dremio image
+        run: docker pull dremio/dremio:latest || echo "Warning: Failed to pre-pull Dremio image, will try at runtime"
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)